writeengine/server/we_ddlcommandproc.cpp

```diff
@@ -59,10 +59,8 @@ WE_DDLCommandProc::WE_DDLCommandProc()
     if (fpc.length() != 0)
         filesPerColumnPartition = cf->uFromText(fpc);
 
-    string epsf = cf->getConfig("ExtentMap", "ExtentsPerSegmentFile");
-
-    if (epsf.length() != 0)
-        extentsPerSegmentFile = cf->uFromText(epsf);
+    // MCOL-4685: remove the option to set more than 2 extents per file (ExtentsPreSegmentFile).
+    extentsPerSegmentFile = 2;
```

Comment:
We should use the constant defined in dbcon/joblist/columncommand-jl.h if this doesn't make a circular dependency.

Reply:
Since EXTENT_PER_SEGMENT_FILE has a storage type (not a macro) it requires to link semantically different parts of projects (ExeManager responsible for select (execplan)) and DDL (responsible for drop, create and so on)). I've added a variable with storage class to DDL, please let me know, if it's not right, and we need to link it against columncomand.

Reply:
No, its OK.